readme.md

```diff
@@ -117,6 +119,20 @@ Note: executing the command `javaflow Person.java` without the `types.yml` file
 to the program failing due to the `Ssn.java` file not being found. `javaflow` does not look
 for additional files which where not supplied as input.
 
+### Verifiers ###
+
+In case you want to verify that you have specified mappings for all types,
+that the DTOs have getters for all fields,
+or that the type of the fields are same as the return types of their getters,
+`javaflow` offers the possibility to add verifiers.
+
+Available verifiers:
+- `MemberFieldsPresentVerifier`, active by default, verifies that there are no missing types referenced in the list of types.
```

Comment:
I see that this is the same sentence we have used in the javadoc for the class. I think it might need some improvement. I could say that it barely works in the class with the given context (e.g. [this paragraph](https://github.com/havardh/javaflow/blob/master/src/main/java/com/github/havardh/javaflow/phases/verifier/MemberFieldsPresentVerifier.java#L41-L44), and that it references the [method signature](https://github.com/havardh/javaflow/blob/master/src/main/java/com/github/havardh/javaflow/phases/verifier/MemberFieldsPresentVerifier.java#L50)). However, standalone in this readme, I didn't understand what it did. :P 

What do you think about this? _verifies that all types used in the types converted are either (1) built-in java types, (2) overridden by custom type substitution, or (3) included as a type being converted. This ensures that the resulting `flow` file will not contain any references to missing types._

Reply:
That sounds good, I was also slightly unsure of what it does and had to check the implementation.
I will update both the JavaDoc and the README with your proposal soon.

Reply:
Great :+1: